code/backend/batch/utilities/helpers/processors/DocumentProcessorHelper.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from typing import List
+
+from ..AzureBlobStorageClient import AzureBlobStorageClient
+
+from ..Processor import Processor
+from ..ConfigHelper import ConfigHelper
+
+from .ProcessorBase import ProcessorBase
+from ..AzureSearchHelper import AzureSearchHelper
+from ..DocumentLoadingHelper import DocumentLoading
+from ..DocumentChunkingHelper import DocumentChunking
+from ...common.SourceDocument import SourceDocument
+
+logger = logging.getLogger(__name__)
+
+
+class DocumentProcessor(ProcessorBase):
```

Comment:
Naming is hard, but I feel this should capture what it is "IndexProcessor" or "PushVectorizer" (I can't think of a good name!) - but DocumentProcessor is not it. Can we also remove the "Helper" from the filename.

Or Something like PushEmbedder?!?

Reply:
seems like we're taking a document and pushing into VectorStore.
`VectorPusher` ?